src/test/suite/fixtures/recorded/languages/ruby/takeAfterKey.yml

```diff
@@ -0,0 +1,24 @@
+languageId: ruby
+command:
```

Comment:
Going to change all of these `take` to `chuck`

Reply:
Assuming you mean replace "take after key" with "chuck key"? I wouldn't advise using "take after" / "chuck after". It's somewhat of an accident that those work imo

For other tests that use "take" (not "take after"), I'd replace them with "change" rather than "chuck", but it's not critical; fine to leave them if already recorded

Reply:
Sounds good.